build_wasm.sh

```diff
@@ -16,6 +16,10 @@ if test -z "$DONT_BENCHMARK_QUACS"; then
 	FLAGS="$FLAGS --features benchmark"
 fi
 
-git submodule update --init --remote --recursive
-cd src/quacs-rs
+# Update our local dependencies (quacs-rs, quacs-data), or clone if possible
+git -C src/store/data pull || git clone https://github.com/quacs/quacs-data src/store/data
+git -C src/quacs-rs pull || git clone https://github.com/quacs/quacs-rs src/quacs-rs
```

Comment:
`quacs-rs` also has a dependency on `quacs-data`, so there should be a pull/clone for `src/quacs-rs/src/data` too.

Reply:
Nevermind I didn't read the next line